### PR TITLE
v1.3.1: chore: economic calendar endpoint and types

### PR DESCRIPTION
**Pull request by SebastianBoehler**
## Summary
Added support for the economic calendar endpoint and added some types

### CODE REVIEW

**Comment by SebastianBoehler** [comment]
Plz let me know again if I should update the version, for a easier merging process

**Comment by tiagosiebler** [comment]
> Plz let me know again if I should update the version, for a easier merging process

Yes, in every PR please

File: src/rest-client.ts [hunk 1]
```diff
@@ -103,8 +103,14 @@ import {
...
   AmendAlgoOrderResult,
   AlgoOrderRequest,
+  EconomicCalendarData,
+  UnitConvertData,
 } from './types';
 import { ASSET_BILL_TYPE } from './constants';
+import {
+  EconomicCalendarRequest,
+  UnitConvertRequest,
+} from './types/rest/request/public';
```
Comment (tiagosiebler) [inline src/rest-client.ts:113]: Take a look at the index.ts in `./types/rest/request`, if you update it to also export public.ts, you'll be able to import this neatly from `./types`

**Review by tiagosiebler — CHANGES_REQUESTED** [review]
Looks great! Just a tiny nitpick on imports

File: src/rest-client.ts (hunk 1 above)
Comment (SebastianBoehler) [reply src/rest-client.ts:113]: Ready to merge from my side

Comment (tiagosiebler) [reply src/rest-client.ts:113]: Now that public.ts is exported, you can move both `EconomicCalendarRequest` and `UnitConvertRequest` to the import from './types'

**Review by tiagosiebler — APPROVED** [review]
LGTM, thanks!

**Pull request merged**